karma.conf.js

```diff
@@ -19,8 +19,7 @@ module.exports = function(config) {
 			'karma-mocha',
 			'karma-phantomjs-launcher',
 			'karma-webpack',
-			'karma-sinon',
-			'karma-coverage'
+			'karma-sinon'
```

Comment:
Oops, I forgot to revert these changes, they were only meant to be for debugging. Can we revert these please?